libraries/protocol/steem_operations.cpp

```diff
@@ -188,6 +188,17 @@ namespace steemit { namespace protocol {
    void witness_update_operation::validate() const
    {
       validate_account_name( owner );
+
+      if ( has_hardfork )
```

Comment:
The point of the previous change was to determine if there were any invalid operations prior to hardfork 0_1. There were not, so we can safely put the post hardfork logic in the validator without a hardfork check.